### PR TITLE
feat: add better guides for quickstart

### PR DESCRIPTION
**Pull request by dejanzele**

### CODE REVIEW

**Review by dave-gantenbein — CHANGES_REQUESTED** [review]
Can you remove references to Kubecon from this so its more evergreen?

File: README.md [hunk 1]
```diff
@@ -123,33 +118,7 @@ are covered by automated tests and appear to be correct.
...
-If you are editing the API definitions, generate the manifests such as CRs or CRDs using:
-
-```bash
-make manifests
-```
-
-**NOTE:** Run `make --help` for more information on all potential `make` targets
-
-More information can be found via the [Kubebuilder Documentation](https://book.kubebuilder.io/introduction.html)
+which provides a reconcile function responsible for synchronizing resources untile the desired state is reached on the cluster
```
Comment (richscott) [inline README.md:121]: Typo: `untile` should be `until` ?

**Review by richscott — COMMENTED** [review]
I saw a typo, but it's not a deal-breaker. I agree that the KubeCon references should probably be removed. 

Note that Lookout V1 images are no longer being built, so it's good that this PR doesn't update to latest versions of the images. I have a bunch of changes to remove all references to Lookout V1, and update to latest numbered versions of all images, but that can wait until after this PR is merged.

**Review by richscott — DISMISSED** [review]

**Review by dave-gantenbein — APPROVED** [review]
lgtm

**Pull request merged**